src/cli/mod.rs

```diff
@@ -238,6 +238,14 @@ pub fn init() {
 					.forbid_empty_values(true)
 					.help("The logging level for the database server")
 					.value_parser(["warn", "info", "debug", "trace", "full"]),
+			)
+			.arg(
+				Arg::new("no-banner")
+					.env("NO_BANNER")
+					.long("no-banner")
+					.required(false)
+					.takes_value(false)
+					.help("Whether a banner should be outputted"),
```

Comment:
shouldn't this be the opposite? e.g. "Whether to hide the startup banner"

Reply:
And could we change this as @macjuul suggested?

Reply:
Thanks for pointing it out! I have pushed the changes.